bindings/python/pymongocrypt/explicit_encrypter.py

```diff
@@ -147,11 +151,37 @@ def create_data_key(self, kms_provider, master_key=None,
                 encoded_names.append(
                     self.callback.bson_encode({'keyAltName': name}))
 
-        opts = DataKeyOpts(master_key, encoded_names)
+        if key_material is not None:
+            key_material = self.callback.bson_encode({'keyMaterial': key_material})
+
+        opts = DataKeyOpts(master_key, encoded_names, key_material)
         with self.mongocrypt.data_key_context(kms_provider, opts) as ctx:
             key = run_state_machine(ctx, self.callback)
         return self.callback.insert_data_key(key)
 
+    def create_key(self, kms_provider, master_key=None,
```

Comment:
We probably don't need it here since these are private APIs, but we will need them on `ClientEncryption` [for parity with the mongosh interface](https://github.com/mongodb/specifications/blob/master/source/client-side-encryption/client-side-encryption.rst#why-is-there-both-a-createkey-and-a-createdatakey-function-in-clientencryption).